syntaxes/Dafny.tmLanguage

```diff
@@ -26,17 +26,17 @@
 				</dict>
 			</dict>
 			<key>end</key>
-			<string>:=|:\||;</string>
-			<key>endCaptures</key>
+			<string>:=|:\||;|(?=const|least|greatest|predicate|function|method|lemma|var|type|datatype|newtype)</string>
```

Comment:
There are entries missing in this list.  Here's the one that dafny-mode uses:

```
(defconst dafny-defuns
  '("class" "codatatype" "const" "constructor" "datatype" "function"
    "iterator" "lemma" "method" "newtype" "predicate" "trait" "type"
    "function method" "predicate method" "module"
    "least predicate" "greatest predicate" "least lemma" "greatest lemma"))
```

Reply:
But then why does the regex have `datatype`, `type`, and `newtype`?

Reply:
Good point. I was going to say, one never knows if these won't be part of a class one day but ... nah ok let me add all of them, it's not costlier